oneandone/src/main/java/org/apache/jclouds/oneandone/rest/compute/config/OneAndOneComputeServiceContextModule.java

```diff
@@ -125,4 +126,28 @@ public boolean apply(Server server) {
                  || server.status().percent() != 0);
       }
    }
+
+   @Provides
+   @Singleton
+   @Named(POLL_PREDICATE_SERVER_SUSPENDED)
```

Comment:
why a not using `ComputeServiceProperties.TIMEOUT_NODE_SUSPENDED` so that you can use a quite adopted pattern like in openstack,digitalocean,packet
```
   @Provides
   @com.google.inject.name.Named(TIMEOUT_NODE_TERMINATED)
   protected Predicate<RegionAndId> provideServerTerminatedPredicate(final NovaApi api, ComputeServiceConstants.Timeouts timeouts,
                                                                ComputeServiceConstants.PollPeriod pollPeriod) {
      return retry(new ServerTerminatedPredicate(api), timeouts.nodeTerminated, pollPeriod.pollInitialPeriod,
              pollPeriod.pollMaxPeriod);
   }
```
?

Reply:
@alibazlamit why do we need this `POLL_PREDICATE_SERVER_SUSPENDED` when jclouds has `TIMEOUT_NODE_SUSPENDED` ?

Reply:
I have changed it to `TIMEOUT_NODE_SUSPENDED`